tools/cldr-code/src/main/java/com/ibm/icu/dev/util/ElapsedTimer.java

```diff
@@ -0,0 +1,105 @@
+// © 2016 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+//
+//  ElapsedTimer.java
+//
+//  Created by Steven R. Loomis on 11/11/2005.
+//  Copyright 2005-2012 IBM. All rights reserved.
```

Comment:
Will CLDR now also have to carry the pre-2016 IBM ICU license?
Maybe it does already?

Reply:
Yanked